src/MgvOfferTaking.sol

```diff
@@ -396,7 +396,9 @@ abstract contract MgvOfferTaking is MgvHasOffers {
       require(sor.offerDetail.gasreq() <= gasreq, "mgv/clean/gasreqTooLow");
       require(Tick.unwrap(sor.offer.tick()) == tick, "mgv/clean/tickMismatch");
 
-      emit OrderStart();
+      // FIXME: Not sure what events we need for cleaning? Maybe none?
```

Comment:
I think that without the cleaning-related events you can't reconstruct the book state?

Reply:
We indeed need the cleaning events :-) At least those that pertain to offers; The comment was targeted specifically at the `OrderStart` and `OrderComplete` events for cleaning - do we need any data on them?

Reply:
Hmm no I don't think so?